traffic_portal/app/src/common/modules/form/server/edit/FormEditServerController.js

```diff
@@ -45,6 +45,7 @@ var FormEditServerController = function(server, $anchorScroll, $scope, $controll
     };
 
     $scope.save = function(server) {
+        console.log(server.profileNames = [server.profileName]);
```

Comment:
This assignment has no return value, so this will only and always ever print `undefined` to the console. It shouldn't be logged.

Reply:
I forgot to remove console.log after i finished with my checks.